flight/PiOS/posix/pios_serial.c

```diff
@@ -80,6 +84,19 @@ static pios_ser_dev * find_ser_dev_by_id(uintptr_t serial)
 	return (pios_ser_dev *) serial;
 }
 
+static void rx_do_cb(pios_ser_dev *ser_dev, uint8_t *incoming_buffer,
```

Comment:
To me that seems ambiguous -- noun vs verb.  and the most-correct rx_invoke_callback seems unwieldly.